config/ubuntu_constants.go

```diff
@@ -113,47 +107,3 @@ var cloudArchivePackagesUbuntu = map[string]struct{}{
 	"uvtool":                  {},
 	"yui3":                    {},
 }
-
-// ubuntuToCentOSPackageNameMap is a map for converting package names from their
-// names in Ubuntu repositories to their equivalent CentOS names.
-var ubuntuToCentOSPackageNameMap = map[string]string{
```

Comment:
This looks like a miss-feature. We probably shouldn't have added this in the first place.